desktop_version/src/main.cpp

```diff
@@ -626,6 +609,7 @@ int main(int argc, char *argv[])
 
     game.init();
     game.seed_use_sdl_getticks = seed_use_sdl_getticks;
+    game.editor_disabled = editor_disabled;
```

Comment:
So as I understand it, as this is the only time that `game.editor_disabled` is assigned, the keyboard must be plugged in on startup to enable the editor, and cannot be plugged in later? Is this desirable behavior?

I don't own a Steam Deck, but I wouldn't find it hard to imagine a situation where a user needs to, for some reason, start the game without a keyboard but then plug it in later. It seems sufficiently not-officially-supported enough that such jank could occur.